src/Commands/ClearFailedJobs.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace ActiveCollab\JobQueue\Commands;
+
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Exception;
+
+/**
+ * @package ActiveCollab\JobQueue\Commands
+ */
+class ClearFailedJobs extends Command
+{
+    /**
+     * Configure command
+     */
+    protected function configure()
+    {
+        parent::configure();
+
+        $this->setName('clear_failed_jobs')
+             ->setDescription('Clear all failed jobs');
+    }
+
+    /**
+     * @param  InputInterface  $input
+     * @param  OutputInterface $output
+     * @return int
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        try {
+            $this->getDatabaseConnection($input)->execute('TRUNCATE TABLE `jobs_queue_failed`');
```

Comment:
Extend `ActiveCollab\JobsQueue\QueueInterface` with `clear()` method and use it instead of direct database clear